src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -990,6 +990,1165 @@ void CmdSketcherConstraint::activated(int /*iMsg*/)
     getSelection().clearSelection();
 }
 
+
+// Dimension tool =======================================================
+
+class DrawSketchHandlerDimension : public DrawSketchHandler
```

Comment:
All methods of this class (including destructor) that overrides methods of the base class should be declared as `override`. The keyword `virtual` is superfluous for these methods and should be removed.

Reply:
Got it